packages/engine/src/physics/classes/Physics.ts

```diff
@@ -151,7 +151,7 @@ function createColliderDesc(
   // if (!colliderDescOptions.shapeType && mesh.geometry.type === 'CylinderGeometry')
   //   colliderDescOptions.shapeType = ShapeType.Cylinder
 
-  if (typeof colliderDescOptions.shapeType === 'undefined') return
+  if (!overrideShapeType && typeof colliderDescOptions.shapeType === 'undefined') return
```

Comment:
is this necessary if we are guaranteed to be passing shapeType in?

Reply:
We are not guaranteed to be passing shapeType in. ColliderDescOptions defines shapeType as possibly undefined